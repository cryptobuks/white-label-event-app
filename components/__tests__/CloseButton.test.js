import React from 'react';
import renderer from 'react-test-renderer';
import { CloseButton } from '../';

describe('CloseButton', () => {
  it('renders correctly', () => {
    const comp = <CloseButton handleGoBack={() => {}} />;

    const tree = renderer.create(comp).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
