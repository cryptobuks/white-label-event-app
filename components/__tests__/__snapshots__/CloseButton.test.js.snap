// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CloseButton renders correctly 1`] = `
<View
  accessibilityComponentType={undefined}
  accessibilityLabel={undefined}
  accessibilityTraits={undefined}
  accessible={true}
  hitSlop={undefined}
  nativeID={undefined}
  onLayout={undefined}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "position": "absolute",
      "right": 4,
    }
  }
  testID={undefined}
>
  <Image
    source={1}
    style={
      Object {
        "height": 24,
        "tintColor": "white",
        "width": 24,
      }
    }
  />
</View>
`;
